src/renderer/Scene.tsx

```diff
@@ -3,10 +3,13 @@ export default class Scene {
   id: Number = 0
   name: string = "Unnamed scene"
   directories: Array<string> = []
-  timingFunction = TF.seconds1
+  timingFunction = TF.constant
+  timingConstant = "1000";
   imageTypeFilter = IF.any
   zoomType = ZF.none
+  zoomLevel: number = 5
   crossFade = false
+  hastebinID: string = "";
```

Comment:
When one of us converts this to URLs instead of hastebin IDs, we need to remember to make the change backward compatible.